packages/documentation/copy/en/reference/ESM Support for Node.md

```diff
@@ -91,11 +87,11 @@ helper();
 This might feel a bit cumbersome at first, but TypeScript tooling like auto-imports and path completion will typically just do this for you.
 
 One other thing to mention is the fact that this applies to `.d.ts` files too.
-When TypeScript finds a `.d.ts` file in package, it is interpreted based on the containing package.
+When TypeScript finds a `.d.ts` file in package, whether it is treated as an ESM or CommonJS file is interpreted based on the containing package.
```

Comment:
I feel like this paragraph really needs to be reworked, but since it was in the experimental docs I’ll do a more minimal fix to try to get this out the door

```suggestion
When TypeScript finds a `.d.ts` file in package, whether it is treated as an ESM or CommonJS file is based on the containing package.
```